src/Init/Data/List/Control.lean

```diff
@@ -71,15 +103,27 @@ def filterAuxM {m : Type → Type v} [Monad m] {α : Type} (f : α → m Bool) :
     let b ← f h
     filterAuxM f t (cond b (h :: acc) acc)
 
+/--
+Applies the monadic predicate `f` on every element in the list, left-to-right, and returns those
+elements `x` for which `f x` returns `true`.
+-/
 @[inline]
 def filterM {m : Type → Type v} [Monad m] {α : Type} (f : α → m Bool) (as : List α) : m (List α) := do
   let as ← filterAuxM f as []
   pure as.reverse
 
+/--
+Applies the monadic predicate `f` on every element in the list, right-to-left, and returns those
+elements `x` for which `f x` returns `true`.
+-/
 @[inline]
 def filterRevM {m : Type → Type v} [Monad m] {α : Type} (f : α → m Bool) (as : List α) : m (List α) :=
   filterAuxM f as.reverse []
```

Comment:
Tip: If possible select all lines that may be affected by a change, then one can use the github suggestion feature to do it.

```suggestion
Applies the monadic predicate `p` on every element in the list, right-to-left, and returns those
elements `x` for which `p x` returns `true`.
-/
@[inline]
def filterRevM {m : Type → Type v} [Monad m] {α : Type} (p : α → m Bool) (as : List α) : m (List α) :=
  filterAuxM p as.reverse []
```